examples/transmissions/create_transmission.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Examples\Transmissions;
+
+require dirname(__FILE__).'/../bootstrap.php';
+
+use SparkPost\SparkPost;
+use GuzzleHttp\Client;
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+
+$httpClient = new GuzzleAdapter(new Client());
+
+/**
+ * configure options in example-options.json
+ */
+$sparky = new SparkPost($httpClient, $options);
+
+$promise = $sparky->transmissions->post([
+	'content' => [
+        'from'=> [
+            'name' => 'SparkPost Team',
+            'email' => 'from@sparkpostbox.com'
+        ],
+        'subject'=>'First Mailing From PHP',
+        'html'=>'<html><body><h1>Congratulations, {{name}}!</h1><p>You just sent your very first mailing!</p></body></html>',
+        'text'=>'Congratulations, {{name}}!! You just sent your very first mailing!'
+    ],
+    'substitution_data'=> ['name'=>'YOUR_FIRST_NAME'],
+    'recipients'=> [
+        [ 'address' => 'YOUR_NAME <YOUR_EMAIL>' ]
```

Comment:
Let's use the expanded name + email form here - I know we have parsing around this but this just pushes them towards using the better format.